tree/ntuple/v7/test/ntuple_view.cxx

```diff
@@ -53,82 +53,25 @@ TEST(RNTuple, View)
    EXPECT_EQ(3, n);
 }
 
-TEST(RNTuple, BulkView)
+TEST(RNTuple, DirectAccessView)
 {
-   FileRaii fileGuard("test_ntuple_bulk_view.root");
+   FileRaii fileGuard("test_ntuple_direct_access_view.root");
 
    auto model = RNTupleModel::Create();
    auto fieldPt = model->MakeField<float>("pt", 42.0);
-   auto eltsPerPage = 10'000;
    {
       RNTupleWriteOptions opt;
-      opt.SetMaxUnzippedPageSize(eltsPerPage * sizeof(float));
-      auto ntuple = RNTupleWriter::Recreate(std::move(model), "myNTuple",
-         fileGuard.GetPath(), opt);
-      for (int i = 0; i < 100'000; i++) {
-         ntuple->Fill();
-      }
-   }
-   auto ntuple = RNTupleReader::Open("myNTuple", fileGuard.GetPath());
-   auto viewPt = ntuple->GetView<float>("pt");
-
-   NTupleSize_t nPageItems = 0;
-   const float *buf = viewPt.MapV(0, nPageItems);
-   ASSERT_EQ(eltsPerPage, nPageItems);
-   for (NTupleSize_t i = 0; i < nPageItems; i++) {
-      ASSERT_EQ(42.0f, buf[i]) << i;
-   }
-   // second last element
-   buf = viewPt.MapV(eltsPerPage - 2, nPageItems);
-   ASSERT_EQ(2, nPageItems);
-   for (NTupleSize_t i = 0; i < nPageItems; i++) {
-      ASSERT_EQ(42.0f, buf[i]) << i;
-   }
-   // last element
-   buf = viewPt.MapV(eltsPerPage - 1, nPageItems);
-   ASSERT_EQ(1, nPageItems);
-   for (NTupleSize_t i = 0; i < nPageItems; i++) {
-      ASSERT_EQ(42.0f, buf[i]) << i;
+      auto writer = RNTupleWriter::Recreate(std::move(model), "myNTuple", fileGuard.GetPath(), opt);
+      writer->Fill();
+      writer->CommitCluster();
+      *fieldPt = 137.0;
+      writer->Fill();
    }
-}
-
-TEST(RNTuple, BulkCollectionView)
```

Comment:
The `BulkView` and `BulkCollectionView` seem to have been removed and not replaced.  Can we expand in the commit log why the test are no longer needed (and/or were now redundant)?

Reply:
Yep, added a comment. Essentially, the tested interface has been replaced by `RFieldBase::RBulk`.